common/lib/xmodule/xmodule/modulestore/split_mongo/split.py

```diff
@@ -1079,6 +1079,12 @@ def get_items(self, course_locator, settings=None, content=None, qualifiers=None
                 For substring matching pass a regex object.
                 For split,
                 you can search by ``edited_by``, ``edited_on`` providing a function testing limits.
+            include_orphans (boolean): To get only those items which are present in the course tree. This gets rid of
+                orphans.
```

Comment:
Needs re-wording.

`Optionally include course orphans in the list. Orphans are items which aren't in the course tree.` or the like...